### ISSUE TITLE
Django + Email Verification Bug

### ISSUE THREAD

**Issue by rdegges** [stormpath/stormpath-django#59]
If you're using Django, and your Stormpath Directory has the Email Verification workflow enabled, it appears that Django isn't respecting these settings (nor is Stormpath).

The account will be instantly enabled in both Stormpath and Django, even before the verification email link has been clicked.

**Comment by mykwillis** [comment]
This may have something to do with the code in models.py `_mirror_data_from_db_user`, which explicitly sets account.status to STATUS_ENABLED if 'is_active' is found in the user model:

    # models.py line 157
        account.status = account.STATUS_DISABLED if data['is_active'] is False else account.STATUS_ENABLED

The StormpathBaseUser object has `is_active` with a default of True:

    # StormpathBaseUser -- models.py line 119
    is_active = models.BooleanField(default=True)


A solution/workaround might be to force `is_active` to False when creating users via Django. 

But to accurately reflect the intent of the Stormpath administrator , `is_active` probably wants to have a default value that depends on whether or not Email Verification is enabled in Stormpath admin.

**Comment by mguidone** [comment]
Same issue here.  Any ETA on when this will included in a release?  I.e. When is the next version bumb?

**Comment by rdegges** [comment]
@avojnovicDk could you prioritize this in your work for this week? We've got some active issues with this and would help quite a few customers =)

**Comment by DanielStevenLewis** [comment]
Has anyone tested that potential workaround?

**Comment by mykwillis** [comment]
I have been setting `user.is_active` to `False` like so:

```Python
    def create(self, validated_data):
        user = User.objects.create(**validated_data)
        user.is_active = False
        user.save()
        return user
```

And it was working fine -- until recently.

Unfortunately, it seems that some recent change has prevented `is_active` from being updated to `True` when the user verifies their email (?).

**Comment by avojnovicDk** [comment]
@rdegges there is PR for this issue: https://github.com/stormpath/stormpath-django/pull/60

**Comment by rdegges** [comment]
Just merged in #60. Will cut release(s) tomorrow.

**Comment by rdegges** [comment]
Closing -- cutting release now.
